reference/constraints/File.rst

```diff
@@ -268,6 +268,38 @@ You can find a list of existing mime types on the `IANA website`_.
     (i.e. the form type is not defined explicitly in the ``->add()`` method of
     the form builder) and when the field doesn't define its own ``accept`` value.
 
+``filenameMaxLength``
+~~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``int`` **default**: ``null``
```

Comment:
```suggestion
**type**: ``integer`` **default**: ``null``
```

Reply:
Thanks!

Reply:
This should be spotted by DOCtor-RST 🤔 , was this the case?

Reply:
Just checked. It was spotted, I just missed it :smile: